items/src/test/scala/weco/api/items/ItemsApiFeatureTest.scala

```diff
@@ -143,42 +206,6 @@ class ItemsApiFeatureTest
       }
     }
 
-    it("returns an empty list if a work has no Sierra-identified items") {
```

Comment:
Why have you removed this test?

Reply:
The test itself is invalid. We no longer filter out non sierra-identified items.

Reply:
Is there a new test that we return them unmodified?